packages/react-components/react-positioning/src/types.ts

```diff
@@ -51,7 +51,17 @@ export type Alignment = 'top' | 'bottom' | 'start' | 'end' | 'center';
 export type AutoSize = 'height' | 'height-always' | 'width' | 'width-always' | 'always' | boolean;
 export type NormalizedAutoSize = { applyMaxWidth: boolean; applyMaxHeight: boolean };
 
-export type Boundary = HTMLElement | Array<HTMLElement> | 'clippingParents' | 'scrollParent' | 'window';
+export type PositioningBoundary =
+  | PositioningRect
```

Comment:
Let's add a test for the rect boundary (popper -> floating UI upgrade still gives me trauma)

Reply:
Added [f5fffa9](https://github.com/microsoft/fluentui/pull/33724/commits/f5fffa93191ac21f38a49d5bc80fe084bfdefb27)